scripts/test-network-stalls.go

```diff
@@ -0,0 +1,248 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
```

Comment:
add `// +build ignore`, because there are multiple scripts in the same folder in the scripts dir

Reply:
The thing that makes life hard with `// +build ignore` is that it makes VSCode ignore it too, and I don't get any of the IDE niceness when working on it. Do you know a workaround, or is that just something one has to deal with in Go land?

Reply:
Yeah, I know :'(... probably the solution would be to have the scripts somehow more nicely grouped and separate folders. I usually remove the `+build ignore` while working on the file, and later add it back.